content/preferences/index.md

```diff
@@ -345,4 +347,4 @@ If you prefer to edit XML instead of using the GUI to modify shortcuts, you may
 
 ### Common Shortcut Mapper Problems
 
-With the introduction of the message area, it is easy to see when a conflict exists between shortcuts.  All you have to do is pick the entry that you _don't_ want to use the conflicted shortcut, and either Clear or Modify the shortcut so there is no longer a conflict.
\ No newline at end of file
+With the introduction of the message area, it is easy to see when a conflict exists between shortcuts.  All you have to do is pick the entry that you _don't_ want to use the conflicted shortcut, and either Clear or Modify the shortcut so there is no longer a conflict.
```

Comment:
Was there a change on (new) line 350?

Reply:
not that I remember.  Maybe a trailing whitespace disappeared or something.